class_defines.py

```diff
@@ -737,7 +737,7 @@ def hide_sketch(self, context):
     target_object: PointerProperty(type=bpy.types.Object)
     curve_resolution: IntProperty(name="Mesh Curve Resolution", default=12, min=1, soft_max=25)
 
-    def dependencies(self):
+    def dependencies(self) -> List[SlvsWorkplane]:
```

Comment:
Probably List[SlvsGenericEntity] makes sense here has the method is implemented on all entity types.

Reply:
Done